client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -60,6 +60,7 @@ interface Props {
 	duplicatePostId: T.PostId;
 	postId: T.PostId;
 	postType: T.PostType;
+	editorType: string;
```

Comment:
```suggestion
	editorType: 'site' | 'post'; // Note: a page other CPT would be a type of post.
```

just an idea to simplify to a kind of enum -- i think that works in typescript?

Reply:
yeah it seems to, that makes sense to me.